### PR TITLE
feat: [QUAD-1742] Add support for zkSync & Base

### PR DESCRIPTION
**Pull request by fabrice-quadrata**
## Motivation
- Update contracts to be zkSync compatible
- Deploy to zkSync Testnet & Mainnet
- Deploy to Base Testnet & Mainnet

*JIRA*: [https://quadrata.atlassian.net/browse/QUAD-<ticket>](https://quadrata.atlassian.net/browse/QUAD-1742)

## Implementation
<_Add any specific details to your implementation._>

## Checklist
- [ ] Unit test implemented 
- [ ] Manual test via `local` or `dev` environment
- [ ] Update technical specifications
- [ ] PR title format follows the guide in the `.github/CONTRIBUTING.md`

### CODE REVIEW

File: contracts/QuadGovernance.sol [hunk 1]
```diff
@@ -77,7 +77,7 @@ contract QuadGovernance is IQuadGovernance, AccessControlUpgradeable, UUPSUpgrad
     /// @param _tokenId tokenId of the passport
     /// @param _uri uri for token
     function setTokenURI(uint256 _tokenId, string memory _uri) external override {
-        require(hasRole(OPERATOR_ROLE, _msgSender()), "INVALID_ADMIN");
+        require(hasRole(OPERATOR_ROLE, _msgSender()) || hasRole(GOVERNANCE_ROLE, _msgSender()), "INVALID_ADMIN");
```
Comment (faze-quadrata) [inline contracts/QuadGovernance.sol:80]: intentional change?

**Review by faze-quadrata — DISMISSED** [review]

File: contracts/QuadGovernance.sol (hunk 1 above)
Comment (fabrice-quadrata) [reply contracts/QuadGovernance.sol:80]: Yes

File: contracts/QuadPassport.sol [hunk 2]
```diff
@@ -93,6 +93,18 @@ contract QuadPassport is IQuadPassport, UUPSUpgradeable, PausableUpgradeable, Qu
...
 
+    /// @notice Claim passport (Mainly used by end-users)
+    /// @param _account Address of the Quadrata Passport holder
+    /// @param _config Input paramters required to set attributes
+    /// @param _sigIssuer ECDSA signature computed by an eligible issuer to authorize the action
+    function claimPassport(
+        address _account,
+        AttributeSetterConfig memory _config,
+        bytes calldata _sigIssuer
+    ) external payable override whenNotPaused {
```
Comment (faze-quadrata) [inline contracts/QuadPassport.sol:104]: Why are we adding this external function?

**Review by faze-quadrata — DISMISSED** [review]

File: contracts/QuadPassport.sol (hunk 2 above)
Comment (fabrice-quadrata) [reply contracts/QuadPassport.sol:104]: We will be moving away from setAttributes & setAttributesBulk to support AA.

So, we'll use SetAttributesIssuer.

I wanted to distinguish the function call between an issuer update and an onboarding process.
When we ever upgrade all contracts on all the chains, we'll update the QuadClient to call claimPassport instead, while leaving the backend continuous monitoring for setAttributesIssuer

**Pull request merged**